lib/comm_common.cpp

```diff
@@ -102,6 +102,13 @@ namespace quda
 
     Topology *topo = new Topology;
 
+#ifdef BUILD_OPENQCD_INTERFACE
+    int *data = static_cast<int *>(map_data);
+    topo->cstar = data[0];
+#else
+    topo->cstar = 0;
+#endif
+
```

Comment:
Here we set `cstar` to the value that comes from openQxD. We set `cstar=0` else, to recover legacy behavior.